palladian-commons/src/test/java/ws/palladian/helper/StopWatchTest.java

```diff
@@ -0,0 +1,192 @@
+
+    PowerMockito.mockStatic(System.class);
+
+    final StopWatch objectUnderTest = new StopWatch();
+    PowerMockito.when(System.currentTimeMillis())
+            .thenReturn(0L)
+            .thenReturn(0L)
+            .thenReturn(0L)
+            .thenReturn(4_758L);
+
+    objectUnderTest.start();
+    objectUnderTest.stop();
+
+    Assert.assertEquals(0L, objectUnderTest.getElapsedTime(false));
+    Assert.assertEquals(0L, objectUnderTest.getElapsedTime(true));
+
+    objectUnderTest.start();
+    objectUnderTest.stop();
+
+    Assert.assertEquals(4_758L, objectUnderTest.getElapsedTime(false));
+    Assert.assertEquals(4L, objectUnderTest.getElapsedTime(true));
+  }
+
+  @PrepareForTest({DateHelper.class, StopWatch.class, System.class})
+  @Test
+  public void testGetElapsedTimeStringAndIncrement()
+          throws Exception {
+
+    PowerMockito.mockStatic(System.class);
+    PowerMockito.mockStatic(DateHelper.class);
+
+    final StopWatch objectUnderTest = new StopWatch();
+    final Method formatDurationMethod =
+            DTUMemberMatcher.method(DateHelper.class, "formatDuration", long.class, long.class);
+    PowerMockito.doReturn("")
+            .when(DateHelper.class, formatDurationMethod)
+            .withArguments(anyLong(), anyLong());
+    final Method formatDurationMethod1 =
+            DTUMemberMatcher.method(DateHelper.class, "formatDuration", long.class);
+    PowerMockito.doReturn(" ")
+            .when(DateHelper.class, formatDurationMethod1)
+            .withArguments(anyLong());
+
+    Assert.assertEquals("  (+)", objectUnderTest.getElapsedTimeStringAndIncrement());
+  }
+
+  @PrepareForTest({DateHelper.class, StopWatch.class, System.class})
+  @Test
+  public void testGetTotalElapsedTimeString()
+          throws Exception {
+
+    PowerMockito.mockStatic(System.class);
+    PowerMockito.mockStatic(DateHelper.class);
+
+    final StopWatch objectUnderTest = new StopWatch();
+
+    final Method formatDurationMethod =
+            DTUMemberMatcher.method(DateHelper.class, "formatDuration", long.class, long.class);
+    PowerMockito.doReturn("")
+            .when(DateHelper.class, formatDurationMethod)
+            .withArguments(anyLong(), anyLong());
+    objectUnderTest.stop();
+
+    Assert.assertEquals("", objectUnderTest.toString());
+    Assert.assertEquals("", objectUnderTest.getTotalElapsedTimeString(false));
+    Assert.assertEquals("", objectUnderTest.getTotalElapsedTimeString(true));
+
+    objectUnderTest.setOutputDetail(null);
+
+    Assert.assertEquals("", objectUnderTest.getTotalElapsedTimeString());
+  }
+
+  @PrepareForTest({DateHelper.class, StopWatch.class, System.class})
+  @Test
+  public void testGetTotalElapsedTimeString2()
+          throws Exception {
+
+    PowerMockito.mockStatic(DateHelper.class);
+
+    final StopWatch objectUnderTest = new StopWatch();
+    final Method formatDurationMethod =
+            DTUMemberMatcher.method(DateHelper.class, "formatDuration", long.class);
```

Comment:
It is possible we might not need to use the Member Matcher from DeeptestUtils, which is one less dependency for them to add (Powermock has its own one, but can be buggy, which is why we wrote our own). Speak to me about how to use powermock's one.